limitador-server/src/envoy_rls/server.rs

```diff
@@ -202,8 +206,14 @@ pub async fn run_envoy_rls_server(
     let rate_limiter = MyRateLimiter::new(limiter, rate_limit_headers);
     let svc = RateLimitServiceServer::new(rate_limiter);
 
+    let reflection_service = tonic_reflection::server::Builder::configure()
+        .register_encoded_file_descriptor_set(rls_proto::RLS_DESCRIPTOR_SET)
+        .build()
+        .unwrap();
+
     Server::builder()
         .add_service(svc)
+        .add_service(reflection_service)
```

Comment:
Thinking we may want to add this when _not_ building for target `release`. We can check for [debug_assertions](https://doc.rust-lang.org/stable/reference/conditional-compilation.html#debug_assertions). Or introduce it as a feature? I guess this isn't meant for a production deployment, right?

Reply:
Yes, makes sense. I will give it a try

Reply:
Implemented as command line option, which defaults to not being exposed.

Reply:
The downside of this implementation is that the build will include `tonic-reflection ` crate now. The feature approach could save us doing that. I think it is handier to have a command line option instead of custom compilation to the reflection service working. Let me know what you think,